app-misc/mosquitto/mosquitto-2.0.14.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="An open source MQTT broker"
+DESCRIPTION="Eclipse Mosquitto is an open source (EPL/EDL licensed) message \
+broker that implements the MQTT protocol versions 5.0, 3.1.1 and 3.1.
+Mosquitto is lightweight and is suitable for use on all devices from low \
+power single board computers to full servers.
+The MQTT protocol provides a lightweight method of carrying out messaging \
+using a publish/subscribe model. This makes it suitable for Internet of Things \
+messaging such as with low power sensors or mobile devices such as phones, \
+embedded computers or microcontrollers."
+HOMEPAGE="http://mosquitto.org"
+COPYRIGHT="2020 Eclipse Foundation."
+LICENSE="EDL"
+REVISION="1"
+SOURCE_URI="https://mosquitto.org/files/source/mosquitto-$portVersion.tar.gz"
+CHECKSUM_SHA256="d0dde8fdb12caf6e2426b4f28081919a2fce3448773bdb8af0d3cd5fe5776925"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

Comment:
Actually I think this should be:  `libVersionCompat="$libVersion compat >= 1"`

Reply:
I agree

Reply:
updated per suggestion